src/style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+html {
```

Comment:
I think that better practice is to set global **font-family** on **body** element as all content lies under body. You can check this discussion what is best practice in this post: https://stackoverflow.com/questions/42925682/where-to-set-font-family-body-or-html-element